packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -120,8 +120,9 @@ export interface IVerticalBarChartProps extends ICartesianChartProps {
 
   /**
    * Specifies the mode of the chart.
+   * @default 'default'
    */
-  mode?: 'plotly';
+  mode?: 'default' | 'plotly';
```

Comment:
can we make this default by default and remove the optional flag.

Reply:
We can set its default value using defaultProps, but removing the optional flag will require users updating to the latest version to explicitly add this prop to vertical bar charts; otherwise, it may throw errors.